lib/Github/Api/Repo.php

```diff
@@ -535,6 +535,25 @@ public function branches($username, $repository, $branch = null, array $paramete
         return $this->get($url, $parameters);
     }
 
+    /**
+     * Sync a fork branch with the upstream repository.
+     *
+     * @link https://docs.github.com/en/rest/branches/branches#sync-a-fork-branch-with-the-upstream-repository
+     *
+     * @param string      $username
+     * @param string      $repository
+     * @param string|null $branchName
+     *
+     * @return array|string
+     */
+    public function mergeUpstream($username, $repository, $branchName = null)
```

Comment:
Can you make this not nullable as it is a required parameter in the API. Also "guessing" the branch name should not be responsibility of this library. We should just pass the value to the github api

Reply:
With default value to make?
$branchName = 'main'

Reply:
I wouldn't provide a default value and just make it required

Reply:
ok